usmqe/api/tendrlapi/cephapi.py

```diff
@@ -18,4 +18,4 @@ def import_cluster(self, nodes, asserts_in=None):
             nodes (list): node list of cluster which will be imported
             asserts_in (dict): assert values for this call and this method
         """
-        super().import_cluster(nodes, "ceph", asserts_in)
+        return super().import_cluster(nodes, "ceph", asserts_in)
```

Comment:
Next time, a fix like this would be better placed in a separate pull request, so that it could be merged quickly.